src/gurt/dlog.c

```diff
@@ -601,10 +613,6 @@ d_log_open(char *tag, int maxfac_hint, int default_mask, int stderr_mask,
 
 	D_INIT_LIST_HEAD(&d_log_caches);
 
```

Comment:
I don't quite understand how is stuff working if you are no longer copying tag into newtag on line 607. Shouldn't you always now do this copy?

Reply:
Hmmm, you are right, I might have done some false testing to validate this ... Sorry and thanks again Alex.